controllers/posts.js

```diff
@@ -61,6 +61,32 @@ const PostsController = {
     });
   },
 
+  Comment: function(req, res) {
+    Post.find({_id: req.params._id}, function(err, post) {
+      console.log(err)
+      console.log("///")
+      console.log(post)
```

Comment:
these console.log can be removed if the implementation is complete

Reply:
changes have been made